websites/R/Raven Code/presence.ts

```diff
@@ -0,0 +1,83 @@
+const presence = new Presence({ clientId: "937015924425367643" }),
+	startTimestamp = Math.floor(Date.now() / 1000);
+let languages: { key: string; page: string; text: string }[] = [];
+
+setInterval(
+	() =>
+		fetch("https://premid.ravencode.live").then(res =>
+			res.text().then(data => (languages = JSON.parse(data)))
```

Comment:
Text? Just await and assign this to the variable
```suggestion
			res.json()
```